<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="cuenta1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>311</width>
     <height>511</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Producto</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cantidad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Precio</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="total1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>550</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TOTAL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="arroz1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ARROZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pera1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PERA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hamburguesa1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>HAMBURGUESA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pizza1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PIZZA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
